packages/plugin-auto-tracking-browser/src/auto-tracking-plugin.ts

```diff
@@ -52,6 +64,15 @@ export const autoTrackingPlugin = (options: Options = {}): BrowserEnrichmentPlug
     if (!element) {
       return false;
     }
+
+    if (shouldTrackEventCallback) {
+      return shouldTrackEventCallback(actionType, element);
```

Comment:
I think "callback" in the name is a little confusing to me. wdyt about `shouldTrackEventResolver` or just `shouldTrackEvent`?

Reply:
Thanks for reviewing! I like the name `shouldTrackEventResolver`, `shouldTrackEvent` sounds like a boolean value, I will update to `shouldTrackEventResolver`.